modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApi.mustache

```diff
@@ -15,23 +15,25 @@ import javax.validation.constraints.*;
 {{#operations}}
 public class {{classname}}ControllerImp {{#useInterfaces}}implements {{classname}}ControllerImpInterface{{/useInterfaces}} {
 
+    {{#examples}}
```

Comment:
Not sure of the meaning of this, I've never understood the concept of "examples" but I don't see any use case where someone would not want the constructor generated. In fact, the mapper has nothing to do with the example. In summary, please explain this, I don't understand :)